tikv/raftstore/config.go

```diff
@@ -193,7 +193,7 @@ func NewDefaultConfig() *Config {
 		ApplyMaxBatchSize:        1024,
 		ApplyPoolSize:            2,
 		StoreMaxBatchSize:        1024,
-		StorePoolSize:            2,
+		StorePoolSize:            1,
```

Comment:
Why change the default value?

Reply:
For benchmark comparison, forgot to change it back, I will make it configurable.